tasties_app/tests/test_ingredient.py

```diff
@@ -0,0 +1,58 @@
+import pytest as pytest
+
+from tasties_app.models import Ingredient, Recipe, Category
+from django.contrib.auth.models import User
+
+
+@pytest.fixture
+def create_data():
+    """
+    Creates 2 users, 2 recipes, and an ingredient,
+    and adds them to the database.
+    """
+    user1 = User.objects.create_user('john', 'password')
+    user2 = User.objects.create_user('paul', 'password')
+    c1 = Category.objects.create(category_name='Breakfast')
+    c1.save()
+    test_data = [
+        ('Test Recipe1', user1, 'Test Description1', 'Test Directions1', 1, 'test_picture1'),
+        ('Test Recipe2', user2, 'Test Description2', 'Test Directions2', 2, 'test_picture2'),
+    ]
+    for title, aid, desc, dirs, mints, pic in test_data:
+        recipe = Recipe(title=title, author_id=aid, description=desc, directions=dirs,
+                        minutes_to_make=mints, recipe_picture=pic)
+        recipe.save()
+        recipe.categories.add(c1)
```

Comment:
Does this operation save the changes to the database (is there now a new Category in the database)?

Reply:
Yes, there is, same as you implemented in the first place.
And yes, this operation saves it to the db.